src/scripts/modules/media/body/content.less

```diff
@@ -340,30 +352,99 @@ figure[data-orient="vertical"] {
+// <p><ol data-display="inline" data-number-style="lower-alpha"><li>item1</li><li>item2</li></ol></p>
+// <p><ol data-display="inline" data-number-style="upper-roman"><li>item1</li><li>item2</li></ol></p>
+// <p><ol data-display="inline" data-number-style="lower-roman"><li>item1</li><li>item2</li></ol></p>
+//
+// <hr/>
+// Inline lists
+// <p>before... <span data-type="list" data-list-type="bulleted" data-bullet-style="myBullet"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="bulleted" data-bullet-style="none"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="bulleted" data-bullet-style="bullet"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="bulleted" data-bullet-style="open-circle"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="bulleted" data-bullet-style="pilcrow"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="bulleted" data-bullet-style="rpilcrow"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="bulleted" data-bullet-style="section"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="bulleted" data-bullet-style="asterisk"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="bulleted" data-bullet-style="dash"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+//
+// <hr/>
+// <p>before... <span data-type="list" data-list-type="enumerated" data-number-style="arabic"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="enumerated" data-number-style="upper-alpha"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="enumerated" data-number-style="lower-alpha"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="enumerated" data-number-style="upper-roman"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+// <p>before... <span data-type="list" data-list-type="enumerated" data-number-style="lower-roman"><span data-type="item">item1</span><span data-type="item">item2</span></span> ...after</p>
+//
+// </div>
+
+
```

Comment:
yikes that's a block of … test stuff.  Since LESS drops `//` comments, this never sees a  browser, right?